doc/release-notes-1.2.0.rst

```diff
@@ -0,0 +1,502 @@
+.. _ncs_release_notes_120:
+
+|NCS| v1.2.0 Release Notes
+##########################
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products.
+It includes the MCUboot and the Zephyr RTOS open source projects, which are continuously integrated and re-distributed with the SDK.
+
+|NCS| v1.2.0 supports product development with the nRF9160 Cellular IoT device.
+It contains reference applications, sample source code, and libraries for Bluetooth Low Energy devices in the nRF52 Series, though product development on these devices is not currently supported with the |NCS|.
+??? What do we say about nRF5340?
+
+
+Highlights
+**********
+
+* Experimental support for the nRF5340 device
+
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v1.2.0**.
+Check the ``west.yml`` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`Getting the nRF Connect SDK code <cloning_the_repositories_win>` for more information.
+
+
+Supported modem firmware
+************************
+
+This version of the |NCS| supports the following modem firmware for cellular IoT applications:
+
+* mfw_nrf9160_1.1.1
+
+Use the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See `Updating the nRF9160 DK cellular modem`_ for instructions.
+
+
+Supported boards
+****************
+
+* PCA10090 (nRF9160 DK)
+* PCA20035 (Thingy:91)
+* PCA10095 (nRF5340 PDK)
+* PCA10056 (nRF52840 Development Kit)
+* PCA10059 (nRF52840 Dongle)
+* PCA10040 (nRF52 Development Kit)
+* PCA10028 (nRF51 Development Kit)
+
```

Comment:
What about support for nrf52811 and nrf52833?

Reply:
PCA10028  should be removed. nRF52811 and 833 is not tested. I am not comfortable in adding it to the supported boards. Agree @bjorn-tore-taraldsen

Reply:
The question is - What do we mean by "Supported boards" here? 
Is it DK's that we have available as a target for our samples or the one we have used during development/ testing?

Reply:
I would claim supported if and only if we have tested. Otherwise how would we know whether it works ? @carlescufi  How  is the claim done in the zephyr world ?

Reply:
There is no claim upstream. All support is "best effort", so not comparable. I would stick to the boards we have actually tested on.

Reply:
@bjorn-tore-taraldsen @shantha-14 Define text to explain what supported means.

Reply:
Leave list as it is now.